.github/renovate.json5

```diff
@@ -3,10 +3,12 @@
     "config:base",
     "schedule:earlyMondays"
   ],
+  includeForks: true,
+  renovateFork: true,
```

Comment:
Why enable it on forks?

Reply:
At the first time to debug renovate on forked repository, these settings might be required. (not sure)